CONTRIBUTING.md

```diff
@@ -155,36 +155,45 @@ To test across clouds, on top of setting up Redis, you also need GCP / AWS / Sno
 > and commenting out tests that are added to `DEFAULT_FULL_REPO_CONFIGS`
 
 **GCP**
-1. Install the [Cloud SDK](https://cloud.google.com/sdk/docs/install).
-2. Then run login to gcloud:
+### Setup your GCP BigQuery Instance
+1. You can get free credits [here](https://cloud.google.com/free/docs/free-cloud-features#free-trial).
+2. You will need to setup a service account, enable the BigQuery api, and create a staging location for a bucket.
```

Comment:
nit: API instead of api

and below: GCP instead of gcp

Reply:
fixed